PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
+
+def perilEndorsement():
+    print "Getting authorization token."
+
+    #URL = "https://api.develop.socotra.com/account/authenticate"
+    URL = "http://127.0.0.1:8080/account/authenticate"
```

Comment:
PEP8 doesn't mandate that you use single or double quotes, but we should probably standardize on one or the other across the company. My personal preference is single quotes and that seems to be what we're using more often than not elsewhere.